### DISCUSSION TITLE
User current reports

### DISCUSSION THREAD

**Discussion by penmat** [jaybo/deepzoom-blog#31] [Ideas]
Current data is fairly coarse grained. What if users could report the actual current? 

There are many places where there isn't any current information, and sometimes knowing the current for one side of a channel vs another would be very useful.

Reliablity might be an issue, but anyone interested enough to report current is probably going to do their best. And over time it would smooth out.

**Comment by penmat** [comment]
There are a couple of ways you could use it:

1. In the early days, maybe just have an indicator to tap to read report (at x date with x sort of tide user reports losing .5 knot)
2. With enough data, start to correlate it with tides and display it - maybe as a "hint" or something?

**Comment by jaybo** [comment]
There's no easy path to adding user defined tide/current predictions.  Al of this data is sourced in a format of Fourier harmonics which are then preprocessed on a server to create the predictions displayed by DeepZoom.  About the only option I can offer is the ability to create "Info" public markers at locations where there are issues with accuracy.

www.deepzoom.com/add-public-markers
